dive_core/generatePm4Info_adreno.py

```diff
@@ -1120,19 +1120,19 @@ def outputFunctionsCpp(pm4_info_file):
   pm4_info_file_h.close();
 
   # lint
-  print("formatting " + "clang-format -i -style=file " + sys.argv[2] + ".h")
-  os.system("clang-format -i -style=file " + sys.argv[2] + ".h")
-  print("formatting " + "clang-format -i -style=file " + sys.argv[2] + ".cpp")
-  os.system("clang-format -i -style=file  " + sys.argv[2] + ".cpp")
+  print('formatting ' + 'clang-format -i -style=file ' + sys.argv[2] + '.h')
+  os.system('clang-format -i -style=file ' + sys.argv[2] + '.h')
+  print('formatting ' + 'clang-format -i -style=file ' + sys.argv[2] + '.cpp')
+  os.system('clang-format -i -style=file  ' + sys.argv[2] + '.cpp')
```

Comment:
nit: could store the clang-format command in a variable to avoid repetition

```suggestion
  format_cmd = 'clang-format -i -style=file ' + sys.argv[2] + '.cpp'
  print('formatting ' + format_cmd )
  os.system(format_cmd)
```

Or, if you want to bump the python min req to 3.6, you could use f strings:

```suggestion
  format_cmd = f'clang-format -i -style=file {sys.argv[2]}.cpp'
  print(f'formatting {format_cmd}')
  os.system(format_cmd)
```

Reply:
fixed